cli/tools/fmt.rs

```diff
@@ -316,6 +324,21 @@ pub fn format_json(
   dprint_plugin_json::format_text(file_path, file_text, &config)
 }
 
+pub fn format_html(
+  _file_path: &Path,
+  file_text: &str,
+  _fmt_options: &FmtOptionsConfig,
+) -> Result<Option<String>, AnyError> {
+  markup_fmt::format_text(
+    file_text,
+    markup_fmt::Language::Html,
+    &Default::default(),
+    |_, code, _| Ok::<_, std::convert::Infallible>(code.into()),
```

Comment:
Handle this more gracefully

Reply:
This last parameter can be used to format CSS or JS/TS.